perftracker/templates/comparison_id.html

```diff
@@ -364,7 +465,74 @@ <h4>Job: {{ job }}</h4>
     $(window).on('DOMContentLoaded load resize scroll', delayed_render);
     $('.collapse').on("shown.bs.collapse hidden.bs.collapse", delayed_render);
     delayed_render();
+
+    var keyUpStack = [];
+
+    $(".panel__input-search_text").keyup(function(e) {
+        keyUpStack.push(1);
+        $('h4').unmark();
+
+        setTimeout(function() {
+            keyUpStack.pop();
+
+            if(keyUpStack.length === 0) {
+                var buf = this.value;
+
+                var filteredList = {};
+                var plotsToShow = {};
+                Object.keys(plotsTitles).forEach(function(key) {
+                    if(!plotsTitles[key].replace(/\s/g, '').includes(buf.replace(/\s/g, ''))) {
```

Comment:
plotsTitles can already have all replacements done in the beginning of the scrtipt.
buf can have all replacements done before the loop.